web/src/app.js

```diff
@@ -14,14 +14,20 @@ import reducer from './reducers';
 import Root from './components/Root';
 import saveMiddleware from './saveMiddleware';
 import { browserIsRed } from './util/browser';
-import { html } from './components/UpgradeBrowser';
+import { html as browserHtml } from './components/UpgradeBrowser';
+import cookieHtml from './components/EnableCookies';
 
 if (browserIsRed) {
   // For browsers we don't support at all, use native DOM APIs to add the
   // warning box since we can't be certain that React will work.
   document.getElementById('app').innerHTML = `
   <div style="margin: 30px;">
-    <div class="ds-col-4 ds-c-alert ds-c-alert--error">${html}</div>
+    <div class="ds-col-4 ds-c-alert ds-c-alert--error">${browserHtml}</div>
+  </div>`;
+} else if (!navigator.cookieEnabled) {
+  document.getElementById('app').innerHTML = `
+  <div style="margin: 30px;">
+    <div class="ds-col-4 ds-c-alert ds-c-alert--error">${cookieHtml}</div>
```

Comment:
Manually keeping both `web/src/app.js` and `web/src/app.dev.js` up-to-date is not ideal. Unsure if we have an issue/ticket to address this.